src/layout/sidebar/menu/Menu.tsx

```diff
@@ -22,19 +24,33 @@ function getItem(
   } as MenuItem
 }
 
-const MainMenu = () => {
+const LanguageToggle = () => {
```

Comment:
can i ask you pls - to refactor this component to separate file?

Reply:
Sure, you're right. I'll rearrange it later